Cargo.toml

```diff
@@ -18,7 +18,7 @@ all-features = true
 grpcio-sys = { path = "grpc-sys", version = "0.2.1" }
 libc = "0.2"
 futures = "^0.1.15"
-protobuf = { version = "1.6", optional = true }
+protobuf = { version = "~2.0", optional = true }
```

Comment:
Why add `~`?

Reply:
`~` limits the range to `>= 2.0.0 < 2.1.0`. `2.0` is the same as `^2.0`, which is `>=2.0.0 < 3.0.0`. Seemed like the smaller version range was preferable to prevent this from being an issue again. I can change it back to just `= "2.0"` if you would prefer